src/main/java/rx/Completable.java

```diff
@@ -1843,7 +1846,7 @@ public final Completable startWith(Completable other) {
      */
     public final Subscription subscribe() {
         final MultipleAssignmentSubscription mad = new MultipleAssignmentSubscription();
-        subscribe(new CompletableSubscriber() {
+        unsafeSubscribe(new CompletableSubscriber() {
```

Comment:
`SafeCompletableSubscriber` is not used here nor the others `subscribe` methods, except for `subscribe(Subscriber)`. I think `subscribe` methods should call between themselves (as in `Observable#subscribe` methods) to refactor code and avoid forgetting this kind of things.